amazon-kinesis-client/src/main/proto/messages.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto2";
```

Comment:
Do we have a mechanism for keeping this in line with the kpl version? I just want to make sure it isn't duplicated somewhere else.

Reply:
Ideally, yes- proto messages would be defined in a separate repo that would then be used as a submodule here and elsewhere. But from what I've seen these definitions are duplicated across multiple repos so consolidating them should be a separate effort.